<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BatchRename</class>
 <widget class="QMainWindow" name="BatchRename">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>802</width>
    <height>534</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BatchRename</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QListWidget" name="lwt_Dir">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>151</width>
      <height>351</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_Menu">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;当前目录:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_Replace">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>460</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>替换</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_Add">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>160</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>添加选中</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_Remove">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>210</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>移除选中</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_Clear">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>260</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>移除全部</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_Root">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>20</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>根目录</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_Up">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>20</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>上一级</string>
    </property>
   </widget>
   <widget class="QListWidget" name="lwt_Work_File">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>60</y>
      <width>281</width>
      <height>351</height>
     </rect>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::ExtendedSelection</enum>
    </property>
   </widget>
   <widget class="QListWidget" name="lwt_File">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>60</y>
      <width>261</width>
      <height>351</height>
     </rect>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::ExtendedSelection</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_AddAll">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>110</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>添加全部</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="let_Addr">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>19</y>
      <width>361</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_Menu_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>430</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;查找的字符串&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_Menu_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>460</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;替换的字符串&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="let_Find">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>430</y>
      <width>361</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="let_Replace">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>460</y>
      <width>361</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>802</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
